src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+    private final double defaultRadius = 10;
+    private final String defaultColor = Color.WHITE.name();
+    private final Figure defaultFigure = new Circle(defaultColor, defaultRadius);
```

Comment:
If you have some default variables which will no change? better save it in constant, for default figure you can create own method